prime-router/src/main/kotlin/fhirengine/azure/FHIRFlowFunctions.kt

```diff
@@ -54,8 +62,15 @@ class FHIRFlowFunctions : Logging {
             responseBuilder.body(body)
         } catch (e: IllegalArgumentException) {
             responseBuilder.body(e.message)
+            actionHistory.trackLogs(
+                ActionLog(InvalidReportMessage(e.stackTraceToString()), type = ActionLogLevel.error)
+            )
         }
-        return responseBuilder.build()
+
+        val response = responseBuilder.build()
+        actionHistory.trackActionRequestResponse(request, response)
```

Comment:
This puts the record into the ActionHistory, but the /convert endpoint does not actually save the data. Seems like an oversight.

Reply:
Updated: now saving there too.